openpype/hosts/blender/api/render_lib.py

```diff
@@ -59,7 +59,7 @@ def get_render_product(output_path, name, aov_sep):
         instance (pyblish.api.Instance): The instance to publish.
         ext (str): The image format to render.
     """
-    filepath = os.path.join(output_path, name)
+    filepath = output_path / name.relative_to(name.anchor)
```

Comment:
`name.relative_to(name.anchor)` 

This could easily introduce massive paths, right? :) 

The anchor being e.g. `c:/` and the path being `c:/project/asset/work/lighting/render/render.0001.png` basically appending `project/asset/work/lighting/render/render.0001.png` to the output dir?

Or why are we doing it like this?

I imagine we want to make this relative maybe to the render root directory?

Reply:
This actually solves a bug that I missed when testing the PR to use the folder paths instead of the asset name 😅 The problem is that the new asset starts with `/`, so when using `os.path.join` it takes it as an anchor and breaks the path, that's why it is necessary! The path is the same as it was before (which is basically a subdir of the workfile) :)

Reply:
Might the intention be clearer if we do `name.lstrip("/")`